volume/struct.go

```diff
@@ -86,6 +86,12 @@ type Volinfo struct {
 	Bricks   []brick.Brickinfo
 }
 
+// VolAuth represents username and password used by trusted/internal clients
```

Comment:
I started out with intent to make this part of `Volinfo` structure itself and then realised that `Volinfo` struct is unmarshalled and sent "as is" to client as json response for `GET` request on the volume. We certainly wouldn't want these auth details to be exposed in client request responses